tests/run-pass/clock.rs

```diff
@@ -0,0 +1,7 @@
+// compile-flags: -Zmiri-disable-isolation
+
+use std::time::SystemTime;
+
+fn main() {
+   let _now = SystemTime::now();
```

Comment:
Will this test work on Windows?

Reply:
Nop, let me add the ignore comment

Reply:
And then please also open an issue tot rack adding Windows support (similar to what we have for env vars).

Reply:
Sure! I'll try to add some instructions if I find the related foreign function